src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -400,6 +401,101 @@ private void updatePresenceStatusForXmppProviders()
+                ProtocolNames.JABBER);
+        AccountManager accountManager
+            = ProtocolProviderActivator.getAccountManager();
+
+        // extract the properties, make sure we decode the password
+        HashMap<String, String> properties = new HashMap<>();
+        propertiesSet.stream().forEach(e ->
+        {
+            String property = e.getKey();
+            String value = e.getValue();
+            if (ProtocolProviderFactory.PASSWORD.equals(property))
+            {
+                value = new String(Base64.decode(value));
+            }
+
+            properties.put(property, value);
+        });
+
+        AccountID xmppAccount
+            = xmppProviderFactory.createAccount(properties);
+
+        // A small workaround to make sure we use the id for the
+        // account that is provided, this is in case we want to
+        // delete the account by the same id.
+        // We use the fact that account manager will reuse the id if found
+        // to store the properties (like edit of an account).
+        {
+            String accountConfigPrefix = accountManager
+                .getFactoryImplPackageName(xmppProviderFactory) + "." + id;
+
+            config.setProperty(accountConfigPrefix, id);
+            config.setProperty(
+                accountConfigPrefix
+                    + "." + ProtocolProviderFactory.ACCOUNT_UID,
+                xmppAccount.getAccountUniqueID());
+        }
+
+        accountManager.loadAccount(xmppAccount);
+
+        logger.info("Added new control muc account:" + id);
+    }
+
+    /**
+     * Removes call control MUC xmmpp account identified by the passed id.
+     * @param id the id of the account to delete.
+     */
+    public static void removeCallControlMucAccount(String id)
+    {
+        ConfigurationService config
+            = JigasiBundleActivator.getConfigurationService();
+        ProtocolProviderFactory xmppProviderFactory
+            = ProtocolProviderFactory.getProtocolProviderFactory(
+                JigasiBundleActivator.osgiContext,
+                ProtocolNames.JABBER);
+        AccountManager accountManager
+            = ProtocolProviderActivator.getAccountManager();
+
+        String accountIDStr = config.getString(
+            accountManager.getFactoryImplPackageName(xmppProviderFactory)
+                + "." + id + "." + ProtocolProviderFactory.ACCOUNT_UID);
+
+        AccountID accountID = accountManager.getStoredAccounts().stream()
+            .filter(a -> a.getAccountUniqueID().equals(accountIDStr))
+            .findFirst().orElse(null);
+
+        if (accountID != null)
+        {
+            boolean result
+                = xmppProviderFactory.uninstallAccount(accountID);
+            logger.info("Removing muc control account: "
+                + id + ", successful:" + result);
+        }
+        else
+        {
+            logger.warn("No muc control account found for removing");
```

Comment:
We should propagate this to HTTP

Reply:
Well ... I kept the same API as in the jvb, as described in the doc. If json is correct 200 OK is returned ... no further output as a result ... but if you say I will change it ...